common/app-starters-task-launch-request-common/README.adoc

```diff
@@ -0,0 +1,56 @@
+=== `Data Flow Task Launch Request Support` Common Module
+
+This artifact contains a Spring Boot auto-configuration providing components to produce a Task Launch Request payload
+. In theory any message source can be used to launch a Task. The task launch request is compatible with the
```

Comment:
Strange formatting to have line starting with the period.
So, far we followed rule *Sentence per line*: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

I can guess that you just have a hard wrapping on some line length.